doc/changes/latest.inc

```diff
@@ -113,8 +113,12 @@ Enhancements
 
 - Add :meth:`mne.time_frequency.EpochsTFR.decimate` to reduce size of time-frequency epochs objects (:gh:`10940` by `Alex Rockhill`_)
 
+- :meth: `mne.coreg.Coregistration.compute_dig_mri_distances` and directly getting distances from mri surfaces now warn user if the distance is greater than 5mm (:gh:`10966` by `Tziona NessAiver`_)
+
 - Add an alias that maps EGI reference channel names (``VREF`` or ``Vertex Reference``) to the electrode name in the standard EGI montages (``Cz``). Now setting standard EGI montages with :meth:`~mne.io.Raw.set_montage` with ``match_alias=True`` will avoid raising an error (:gh:`10823` by `Scott Huberty`_)
 
+- Make it possible to enter sHH:MM timezone information when calling :meth:`mne.channels.set_meas_date` (:gh:`10970` by `Tziona NessAiver`)
```

Comment:
```suggestion
- Make it possible to enter sHH:MM timezone information when calling :meth:`mne.io.Raw.set_meas_date` and related methods (:gh:`10970` by `Tziona NessAiver`)
```

Reply:
... should fix both warnings (treated as errors) in the CircleCI and Azure doc builds